cli/list/list.go

```diff
@@ -0,0 +1,161 @@
+package list
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"sort"
+	"strconv"
+	"strings"
+
+	"github.com/apex/log"
+	"github.com/fatih/color"
+	"github.com/tarantool/tt/cli/cmdcontext"
+	"github.com/tarantool/tt/cli/config"
+	"github.com/tarantool/tt/cli/util"
+)
+
+// printBinaries outputs installed versions of the program.
+func printBinaries(program string, binList []string, hasMaster bool, binActive string) {
+	if len(binList) > 0 || hasMaster {
+		log.Infof(program + ":")
+		if hasMaster {
+			if binActive == "master" {
+				fmt.Printf("	%s\n", util.Bold(color.GreenString("master - active")))
+			} else {
+				fmt.Printf("	%s\n", color.YellowString("master"))
+			}
+
+		}
+		for _, bin := range binList {
+			if bin == binActive {
+				fmt.Printf("	%s\n", util.Bold(color.GreenString(bin+" - active")))
+			} else {
+				fmt.Printf("	%s\n", color.YellowString(bin))
+			}
+		}
+	}
+}
+
+// sortBinaries sorts versions of binaries.
+func sortBinaries(binList []string) {
```

Comment:
We have SortVersions function already. Is there something wrong with it?
Why is there "binaries" word in this function name? I do not see executable-specific code in it.

Reply:
I guess I could, but with in my case it will require more code. I don't think it is necessary.

Reply:
It will not require more code. SortBinaries function must be deleted and sorting function from the `version` package msut be reworked to be more flexible and to support `reverse`. So manual loop to reverse versions is not required also.